docs/pages/reference/config.mdx

```diff
@@ -437,22 +437,22 @@ auth_service:
     # (https://goteleport.com/docs/architecture/proxy/#recording-proxy-mode).
     proxy_checks_host_keys: yes
 
-    # Determines if SSH sessions to cluster nodes are forcefully terminated
-    # after no activity from a client (idle client).
+    # Determines if Server and Database sessions to cluster nodes are forcefully
+    # terminated after no activity from a client (idle client).
     # Examples: "30m", "1h" or "1h30m"
     client_idle_timeout: never
 
     # Send a custom message to the client when they are disconnected due to
     # inactivity. The empty string indicates that no message will be sent.
-    # (Currently only supported for SSH connections)
+    # (Currently only supported for Server and Database connections)
     client_idle_timeout_message: ""
```

Comment:
Unfortunately, it looks like even though `client_idle_timeout` works on most of out supported protocols, we only use the message for SSH sessions. This is likely an oversight. Mind creating a separate issue for this?